index.js

```diff
@@ -194,4 +194,23 @@ linkSection[2].addEventListener('click', () => {
   headerNavMobile.classList.add('hidden');
 });
 
-document.addEventListener('DOMContentLoaded', insertCards);
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', insertCards);
+
+const finishform = document.querySelector('.finishForm');
+const email = document.querySelector('.inputEmail');
+const emailError =document.querySelector('.emailError')
+const regExp = /^[a-z0-9]+@[a-z0-9-]+\.[a-z0-9-.]+$/;
+console.log(finishform)
+
+finishform.addEventListener('submit', (e) => {
+    if (email.value.match(regExp)){
+        emailError.style.display = 'none';
+        return true;
+    }else {
+        e.preventDefault();
+        emailError.style.display = 'block';
+        console.log(emailError)
+        return false;
+    }
+})
```

Comment:
- I think this validate contact form milestone requires you to ensure that the contact form contains at least one validation checking if the email is written in all lowercase.

- When I try to fill the form with both upper and lower case, it doesn't submit. I think this is because you haven't implemented the code for checking lower cases.

- This is what this project requires us to do 👉🏽  If a user submits the contact form with the email field containing `uppercase` characters an `error message` should show up.

Can we kindly fix this? 🤗